src/components/navbar/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Navbar.css';
+
+function Navbar () {
+    return (
+
+            <div className ='navi'>
+              <nav>
+              
+              <img alt='nextflic-logo' src='../../../images/NETFLIX.png'></img>
```

Comment:
Don't hardcode image src unless it comes from another server. 

Here, you should put the logo in the `NavBar` folder with the `NavBar.js`. You should then use `import logo from './logo.png';` and `<img src={logo} alt="NextFlick Logo" />`.

`img` elements are self closing : `<img />`, not `<img></img>`.

Lowercase the logo and adjust the name, either `nextflick.png` or `logo.png`. Also, vector files in `.svg` are better suited for logos.

alt attributes should be written in natural language, it's normal text to describe with text what  the image represents.